buildprocess/webpack.config.js

```diff
@@ -8,10 +8,12 @@ var path = require('path');
 
 module.exports = function(devMode, hot) {
     var config = {
-        entry: './index.js',
+        entry: {
+          entry: './entry.js'
+         },
         output: {
             path: path.resolve(__dirname, '..', 'wwwroot', 'build'),
-            filename: 'TerriaMap.js',
+            filename: '[name].bundle.js',
```

Comment:
Can we revert this change? And then we can revert the index.html change too, which will be good because it will potentially avoid conflicts for our users in that file.